test/functionalTest/cases/0000_bad_requests/erroneous_input.test

```diff
@@ -36,7 +36,7 @@ orionCurl --url "/v1/registry/nothingAtAll"
 
 --REGEXPECT--
 0: ++++++++++++++++++++
-HTTP/1.1 200 OK
+HTTP/1.1 400 Bad Request
```

Comment:
This change in .test (and some other similar ones) correspond to the following entry in CNR: "Fix: out-of-NGSI service not found error now using 400 Bad Request HTTP response code (instead of 200 OK). Error message has been unified to "service not found" in all cases."